stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -2186,6 +2217,38 @@ pub mod test {
         make_tx(key, nonce, 0, payload)
     }
 
+    pub fn readonly_call(
```

Comment:
This is a test helper, it should not have been moved to this location. It takes a `TestPeer` and is hardcoded to testnet. 

You can put it back in `signer_tests.rs` or some other test file.

Reply:
done